docs(workflow): document external-config target repo adoption recipe

## Problem

`workflow-check --config <path>` and `spec doctor` external-config capability checks are now implemented, but the downstream adoption contract still needs an explicit recipe for target repos such as tachigo/tachiya:

- where external config can live,
- which `workflow-check` commands should include `--config`,
- which PR body fields should copy only status/ref evidence,
- which `.spec-injector/`, generated output, local JSON, and private context artifacts must not be committed.

Without this recipe, target repos may wire the new capability inconsistently or ask Scope Police to parse full spec evidence.

## Goal

Publish a docs/test adoption recipe for external-config workflow-check use in downstream target repos.

## Scope

Allowed files:
- `docs/target-repo-adoption-contract.md`
- `docs/ai-bootstrap-install-contract.md`
- `README.md` only for a short link/note if needed
- focused docs link/assertion tests, likely `tests/docs-links.test.ts` or existing docs contract tests

Expected behavior:
- Docs show `workflow-check --config <external-config>` usage for start/commit/merge local gates.
- Docs state downstream PR bodies/ledgers copy status/ref evidence only, not full spec plan/task-package output.
- Docs state target repos must not commit `.spec-injector/`, `.spec-injector/out/`, generated output, local routing/readback JSON, private context, or private ledgers.
- Tests assert the adoption contract remains linked and contains the external-config recipe and no-mutation/no-artifact boundary.

## AWP routing requirement

Use Hybrid AWP with explicit fallback gate:
- routing_mode: hybrid_awp
- routing_task_class: docs_contract_slice
- dispatch a bounded worker/reviewer, or record explicit controller-direct fallback before implementation.

## Non-goals

- Do not change runtime CLI behavior.
- Do not change downstream repo Scope Police.
- Do not add hosted control plane, daemon, dashboard, auto-comment, auto-merge, or hidden worker runtime.
- Do not mutate target repos.

--- a/docs/target-repo-adoption-contract.md
+++ b/docs/target-repo-adoption-contract.md
@@ -14,6 +14,33 @@ Target repos can use the published CLI behavior directly when `spec-injector` is
 - Offline/manual checklist fallback remains valid for repos that have not fully adopted `spec-injector`.
 - Target repo PR bodies only need stable status/ref evidence, not full task packages or full AWP review ledgers.
 
+## External Config Recipe
+
+Target repos do not need to commit `.spec-injector/` just to use `workflow-check`. If a repo keeps its spec config in a shared tooling checkout, temporary bootstrap directory, or other approved location outside the target repo, pass that snapshot explicitly:
+
+```bash
+spec workflow-check --repo . --config <external-config> --phase start --issue <number-or-url> --format json
+spec workflow-check --repo . --config <external-config> --phase commit --pr-body /path/to/pr-body.md --format json
+spec workflow-check --repo . --config <external-config> --phase merge --pr-body /path/to/pr-body.md --head-sha <sha> --readback-evidence /path/to/readback.json --format json
+```
+
+Use the external config for local start / commit / merge PR-body gates when the target repo intentionally does not contain a committed `.spec-injector/config.json`. `merge --pr` readback can still run without local config because it reads PR metadata, checks, reviews, and PR body through read-only helpers; if `--config <external-config>` is provided, the path must exist and validate.
+
+Downstream PR bodies and ledgers should copy status/ref evidence only:
+
+- `spec gate status`
+- `spec evidence ref`
+- `routing evidence status`
+- `routing evidence ref`
+- `finding disposition status`
+- `finding disposition ref`
+- `threshold evidence status`
+- `threshold ledger ref`
+- `latest head SHA`
+- `ready_to_merge`
+
+Keep the full `spec plan` output, task packages, routing ledgers, readback JSON, review transcripts, and private context behind the referenced evidence location. Scope Police should validate the thin status/ref surface and must not parse private or generated spec evidence.
+
 ## When Target Repo PRs Are Still Needed
 
 Downstream repos still need their own PR when they want to change repo-local policy or enforcement:
@@ -28,7 +55,8 @@ Downstream repos still need their own PR when they want to change repo-local pol
 
 ## Non-Negotiable Boundaries
 
-- Do not commit `.spec-injector/out/`, generated task packages, local routing JSON, private context, or private ledgers into target repos.
+- Do not commit `.spec-injector/` into target repos when using an external config snapshot.
+- Do not commit `.spec-injector/out/`, generated task packages, local routing/readback JSON, private context, or private ledgers into target repos.
 - Do not make target repo Scope Police parse full `spec plan` output, full task packages, or full AWP review ledgers.
 - Do not use `spec-injector` as a hosted control plane, daemon, dashboard, merge bot, auto-commenter, or hidden LLM wrapper.
 - Do not treat checker `pass` as human approval.

--- a/tests/hybrid-awp-routing-policy.test.ts
+++ b/tests/hybrid-awp-routing-policy.test.ts
@@ -139,7 +139,13 @@ test('target repo adoption contract remains linked from workflow docs', async ()
   assert.match(contract, /Target Repo Adoption Contract/);
   assert.match(contract, /tachigo/i);
   assert.match(contract, /tachiya/i);
+  assert.match(contract, /workflow-check --repo \. --config <external-config>/);
+  assert.match(contract, /external config/i);
+  assert.match(contract, /status\/ref evidence only/i);
+  assert.match(contract, /local routing\/readback JSON/);
   assert.match(contract, /Do not commit `.spec-injector\/out\/`/);
+  assert.match(contract, /Do not commit `.spec-injector\/`/);
+  assert.match(contract, /private ledgers/);
   assert.match(contract, /does not mutate downstream repos/i);
   assert.match(workflow, /\[target repo adoption contract\]\(target-repo-adoption-contract\.md\)/);
 });

